examples/step-71/doc/intro.dox

```diff
@@ -0,0 +1,488 @@
+using integrals over the entire domain (i.e., the union of all cells),
+one would end up with two derivatives on the test functions and trial
+functions each. If one were to use the usual piecewise polynomial
+functions with their kinks on cell interfaces, the first derivative
+would yield a discontinuous gradient, and the second derivative with
+delta functions on the interfaces -- but because both the second
+derivatives of the test functions and of the trial functions yield a
+delta function, we would try to integrate the product of two delta
+functions. For example, in 1d, where $\varphi_i$ are the usual
+piecewise linear "hat functions", we would get integrals of the sort
+@f{align*}{
+  \int_0^L (\Delta \varphi_i) (\Delta \varphi_j)
+  =
+  \int_0^L
+  \frac 1h \right[\delta(x-x_{i-1}) - 2\delta(x-x_i) + \delta(x-x_{i+1})\right]
+  \frac 1h \right[\delta(x-x_{j-1}) - 2\delta(x-x_j) + \delta(x-x_{j+1})\right]
+@f}
+where $x_i$ is the node location at which the shape function
+$\varphi_i$ is defined, and $h$ is the mesh size (assumed
+uniform). The problem is that delta functions in integrals are defined
+using the relationship
+@f{align*}{
+  \int_0^L \delta(x-\hat x) f(x) \; dx
+  =
+  f(\hat x).
+@f}
+But that only works if (i) $f(\cdot)$ is actually well defined at
+$\hat x$, and (ii) if it is finite. On the other hand, an integral of
+the form
+@f{align*}{
+\int_0^L \delta(x-x_i) \delta (x-x_i)
+@f}
+does not make sense. Similar reasoning can be applied for 2d and 3d
+situations.
+
+In other words: This approach of trying to integrate over the entire
+domain and then integrating by parts can't work.
+
+Historically, numerical analysts have tried to address this by
+inventing finite elements that are "$C^1$ continuous", i.e., that use
+shape functions that are not just continuous but also have continuous
+first derivatives. This is the realm of elements such as the Argyris
+element, the Clough-Tocher element and others, all developed in the
+late 1960s. From a twenty-first century perspective, they can only be
+described as bizarre in their construction. They are also exceedingly
+cumbersome to implement if one wants to use general meshes. As a
+consequence, they have largely fallen out of favor and deal.II currently
+does not contain implementations of these shape functions.
+
+
+<h2> What to do instead? </h2>
+
+So how does one approach solving such problems then? That depends a
+bit on the boundary conditions. If one has the first set of boundary
+conditions, i.e., if the equation is
+@f{align*}{
+  \Delta^2 u(\mathbf x) &= f(\mathbf x)
+  \qquad \qquad &&\forall \mathbf x \in \Omega, \\
+  u(\mathbf x) &= g(\mathbf x) \qquad \qquad
+  &&\forall \mathbf x \in \partial\Omega, \\
+  \Delta u(\mathbf x) &= h(\mathbf x) \qquad \qquad
+  &&\forall \mathbf x \in \partial\Omega,
+@f}
+then the following trick works (at least if the domain is convex, see
+below): In much the same as we obtained the
```

Comment:
Is your comment independent of the change request? If yes, can you clarify what's unclear to you?

Reply:
I should have been clearer: I just didn't understand the fragment "In much the same we obtained", it sounds like its missing a word to me. The suggested change fixes my issue.

Reply:
I should have been clearer. The original version of that sentence was confusing to me because it looked like it was missing a word. The corrected version makes sense to me.